extensions/2.0/Khronos/KHR_techniques_webgl/README.md

```diff
@@ -0,0 +1,836 @@
+
+* **Type**: `integer`
+* **Required**: No
+* **Minimum**: ` >= 0`
+
+### uniform.type :white_check_mark: 
+
+The datatype.  All valid values correspond to WebGL enums.
+
+* **Type**: `integer`
+* **Required**: Yes
+* **Allowed values**:
+   * `5120` BYTE
+   * `5121` UNSIGNED_BYTE
+   * `5122` SHORT
+   * `5123` UNSIGNED_SHORT
+   * `5124` INT
+   * `5125` UNSIGNED_INT
+   * `5126` FLOAT
+   * `35664` FLOAT_VEC2
+   * `35665` FLOAT_VEC3
+   * `35666` FLOAT_VEC4
+   * `35667` INT_VEC2
+   * `35668` INT_VEC3
+   * `35669` INT_VEC4
+   * `35670` BOOL
+   * `35671` BOOL_VEC2
+   * `35672` BOOL_VEC3
+   * `35673` BOOL_VEC4
+   * `35674` FLOAT_MAT2
+   * `35675` FLOAT_MAT3
+   * `35676` FLOAT_MAT4
+   * `35678` SAMPLER_2D
+
+### uniform.semantic
+
+Identifies a uniform with a well-known meaning.  Uniform semantics include `"LOCAL"` (`FLOAT_MAT4`), `"MODEL"` (FLOAT_MAT4), `"VIEW"` (FLOAT_MAT4), `"PROJECTION"` (FLOAT_MAT4), `"MODELVIEW"` (FLOAT_MAT4), `"MODELVIEWPROJECTION"` (FLOAT_MAT4), `"MODELINVERSE"` (FLOAT_MAT4), `"VIEWINVERSE"` (FLOAT_MAT4), `"PROJECTIONINVERSE"` (FLOAT_MAT4), `"MODELVIEWINVERSE"` (FLOAT_MAT4), `"MODELVIEWPROJECTIONINVERSE"` (FLOAT_MAT4), `"MODELINVERSETRANSPOSE"` (FLOAT_MAT3), `"MODELVIEWINVERSETRANSPOSE"` (FLOAT_MAT3), `"VIEWPORT"` (FLOAT_VEC4), `"JOINTMATRIX"` (FLOAT_MAT4[]).  For forward-compatibility, application-specific semantics must start with an underscore, e.g., `"_SIMULATION_TIME"`.
+
+* **Type**: `string`
+* **Required**: No
+
+### uniform.value
+
+The value of the uniform. The length is determined by the values of the `type` and `count` (if present) properties.  A material uniform value with the same name, when specified, overrides this value.
+
+* **Type**: `any`
+* **Required**: No
+
+### uniform.name
+
+* **Type**: `any`
+* **Required**: No
+
+### uniform.extensions
+
+Dictionary object with extension-specific objects.
+
+* **Type**: `object`
+* **Required**: No
+* **Type of each property**: Extension
+
+### uniform.extras
+
+Application-specific data.
+
+* **Type**: `any`
+* **Required**: No
+
+---------------------------------------
+<a name="reference-khr_techniques_webgl-material-extension"></a>
+## KHR_techniques_webgl material extension
+
+The technique to use for a material and any additional uniform values.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**technique**|`integer`|The index of the technique.| :white_check_mark: Yes|
+|**values**|`object`|Dictionary object of uniform values.|No, default: `{}`|
+|**extensions**|`object`|Dictionary object with extension-specific objects.|No|
+|**extras**|`any`|Application-specific data.|No|
+
+Additional properties are allowed.
+
+* **JSON schema**: [material.KHR_technique_webgl.schema.json](schema/material.KHR_technique_webgl.schema.json)
+
+### khr_techniques_webgl.material.extension.technique :white_check_mark: 
```

Comment:
Is it fine to drop the `khr_techniques_webgl.material.extension.` prefix for these? It looks like [KHR_materials_pbrSpecularGlossiness](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_pbrSpecularGlossiness) doesn't have the prefix.